components/cdc/src/channel.rs

```diff
@@ -416,13 +421,38 @@ impl<'a> Drain {
                 sink.feed((e, write_flags)).await?;
             }
             sink.flush().await?;
+            sleep_after_sink_flush().await;
+            // Update last flush time if provided
+            if let Some(time_tracker) = last_flush_time {
+                use std::time::{SystemTime, UNIX_EPOCH};
```

Comment:
It's better to place `use` statements at the top of the function or module rather than inside a block.